package.json

```diff
@@ -1,12 +1,22 @@
 {
   "name": "@itwin/reality-data-client",
-  "version": "1.0.0",
-  "description": "Reality Data Client for the iTwin platform",
+  "version": "0.9.0",
```

Comment:
Why are we going back to 0.9? Is this a bad merge?

Reply:
i'm introducing beachball in this pr to help manage versions and changelogs.
in the publish pipeline, beachball will bump the version based on the changelog files generated
in this pr i generated a changefile, and marked it as `major`, so when the pipeline runs, itll publish as 1.0

Reply:
Ok, make sense.